cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/preview/DefaultPreviewStore.java

```diff
@@ -86,15 +86,18 @@
 
   private final AtomicLong counter = new AtomicLong(0L);
 
-  private final LevelDBTableCore table;
+  private final LevelDBTableCore dataAndMdTable;
```

Comment:
so this one stores preview data? Maybe better call it `previewDataTable`?

Reply:
It stores both preview data and preview run status like start, end time etc, so more than just preview data.

Reply:
So, maybe call it `previewTable` and `previewQueueTable`?

Reply:
Done.